scheduler/src/cook/mesos/api.clj

```diff
@@ -1737,14 +1737,15 @@
 
 ;; /share and /quota
 (def UserParam {:user s/Str})
+(def ReasonParam {:reason s/Str})
+(def UserLimitDestroyParams (merge UserParam ReasonParam))
 
 (def UserLimitsResponse
   {String s/Num})
 
 (defn set-limit-params
   [limit-type]
-  {:body-params (merge UserParam {limit-type {s/Keyword s/Num}
-                                  :reason s/Str})})
+  {:body-params (merge UserParam ReasonParam {limit-type {s/Keyword s/Num}})})
```

Comment:
Should we use `UserLimitDestroyParams` here as well?

Reply:
I was thinking that too, but the name didn't fit, and I didn't like "UserAndReason" as an identifier... I just came up with a better name though: `UserLimitChangeParams`